graphql/resolvers.go

```diff
@@ -1164,22 +1165,20 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, execution
 	if dbTask == nil || err != nil {
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
 	}
-	baseTask, err := dbTask.FindTaskOnBaseCommit()
-	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding base task with id %s: %s", taskID, err))
+	opts := apimodels.GetCedarTestResultsOptions{
+		BaseURL:   evergreen.GetEnvironment().Settings().Cedar.BaseURL,
+		Execution: *execution,
 	}
-
-	var taskExecution int
-	taskExecution = dbTask.Execution
-
-	baseTestStatusMap := make(map[string]string)
-	if baseTask != nil {
-		baseTestResults, _ := r.sc.FindTestsByTaskId(baseTask.Id, "", "", "", 0, taskExecution)
-		for _, t := range baseTestResults {
-			baseTestStatusMap[t.TestFile] = t.Status
-		}
+	if len(dbTask.ExecutionTasks) > 0 {
+		opts.DisplayTaskID = taskID
+	} else {
+		opts.TaskID = taskID
 	}
 
+	cedarTestResults, err := apimodels.GetCedarTestResults(ctx, opts)
+	if err != nil {
+		grip.Warning(fmt.Sprintf("Did not find cedar test results for task: %s", taskID))
```

Comment:
Sorry for being annoying but can you do something like this:

```
		grip.Warning(message.WrapError(err, message.Fields{
			"task_id": taskID,
			"message": "problem getting cedar test results",
		}))
```

Reply:
Done